sdk/storage/storage-file-share/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+## 12.1.0 (2020-02-11)
+- Updated Azure Storage Service API version to 2019-07-07.
+- A new type `ShareLeaseClient` is added to manage leases.
+- New SMB parameters for file copy.
+- Force Close Handles Response now includes the number of handles that failed to close.
```

Comment:
minor: what about directly use class name for Force Close Handles Respons

Reply:
We have both low level and wrapping level interface changes, for force Close Handle and forceCloseAllHandles, so being vague here.